app/auto-eligibility/config/index.js

```diff
@@ -6,9 +6,6 @@ const schema = Joi.object({
     enabled: Joi.bool().default(true),
```

Comment:
General comment - code coverage in SonarCloud is estimated <90% after merge, however I'm not sure this project config is correct as it still appears to pass the check

https://sonarcloud.io/summary/new_code?id=ffc-ahwr-eligibility&pullRequest=67